src/extension/loadLogs.ts

```diff
@@ -33,42 +31,19 @@ export async function loadLogs(uris: Uri[], token?: { isCancellationRequested: b
     logs.forEach(log => Telemetry.sendLogVersion(log.version, log.$schema ?? ''));
     logs.forEach(tryFastUpgradeLog);
 
-    const logsNoUpgrade = [] as Log[];
-    const logsToUpgrade = [] as Log[];
-    const warnUpgradeExtension = logs.some(log => detectUpgrade(log, logsNoUpgrade, logsToUpgrade));
-    const upgrades = logsToUpgrade.length;
-    if (upgrades) {
-        await window.withProgress(
-            { location: ProgressLocation.Notification },
-            async progress => {
-                for (const [i, oldLog] of logsToUpgrade.entries()) {
-                    if (token?.isCancellationRequested) break;
-                    progress.report({
-                        message: `Upgrading ${i + 1} of ${upgrades} log${upgrades === 1 ? '' : 's'}...`,
-                        increment: 1 / upgrades * 100
-                    });
-                    await new Promise(r => setTimeout(r, 0)); // Await otherwise progress does not update. Assumption: await allows the rendering thread to kick in.
-                    const {fsPath} = Uri.parse(oldLog._uri, true);
-                    try {
-                        const tempPath = upgradeLog(fsPath);
-                        const file = fs.readFileSync(tempPath, 'utf8'); // Assume scheme file.
-                        const log = JSON.parse(file) as Log;
-                        log._uri = oldLog._uri;
-                        log._uriUpgraded = Uri.file(tempPath).toString();
-                        logsNoUpgrade.push(log);
-                    } catch (error) {
-                        console.error(error);
-                        window.showErrorMessage(`Failed to upgrade '${fsPath}'`);
-                    }
-                }
-            }
-        );
+    const logsSupported = [] as Log[];
+    const logsNotSupported = [] as Log[];
+    const warnUpgradeExtension = logs.some(log => detectUpgrade(log, logsSupported, logsNotSupported));
+    for (const log of logsNotSupported) {
+        if (token?.isCancellationRequested) break;
+        const {fsPath} = Uri.parse(log._uri, true);
+        window.showWarningMessage(`'${fsPath}' was not loaded. Version '${log.version}' and schema '${log.$schema ?? ''}' is not supported.`);
     }
-    logsNoUpgrade.forEach(log => augmentLog(log, driverlessRules));
+    logsSupported.forEach(log => augmentLog(log, driverlessRules));
     if (warnUpgradeExtension) {
         window.showWarningMessage('Some log versions are newer than this extension.');
     }
-    return logsNoUpgrade;
+    return logsSupported;
```

Comment:
>logsSupported [](start = 11, length = 13)

Would you mind calling it `supportedLogs`?

Reply:
Likewise with logsNotSupported? I might be weird but prefixes (as opposed to suffixes) help my mind group similar things. The visual alignment "clicks" for me.

Reply:
Up to you. In my commented below I suggested `supportedLogs`, `oldVersionLogs`, `unrecognizedSchemaLogs`, and `futureVersionLogs`.  But you're the one that has to read the code every day.

---
In reply to: [483873846](https://github.com/microsoft/sarif-vscode-extension/pull/348#discussion_r483873846) [](ancestors = 483873846)